docs/software/printing.md

```diff
@@ -60,4 +61,24 @@ Website: <https://octoprint.org>
 Official documentation: <https://docs.octoprint.org>  
 Forum: <https://community.octoprint.org>
 
+## CUPS
+
+CUPS provides a web interface for managing your local or network printers.
+
+![CUPS web interface screenshot](../assets/images/dietpi-software-printserver-cups.png){: width="500" height="385" loading="lazy"}
+
+=== "Access the web interface"
+
+    The web interface is accessible via port **631**:
+
+    - URL = `http(s)://<your.IP>:631`  
+    - Username = `root`
+    - Password = `<RootPassword>`
```

Comment:
Harmonization: `<your global password>`

Reply:
It's indeed the root password and not the global application pw